cmd/maya-apiserver/app/command/start.go

```diff
@@ -126,7 +127,12 @@ func Run(cmd *cobra.Command, c *CmdStartOptions) error {
 	if mconfig == nil {
 		return errors.New("Unable to load the configuration.")
 	}
-
+	go func(){
+		err := spc.Start()
+		if err != nil{
+			glog.Errorf("coould not start controller: %s", err.Error())
+		}
+	}()
 	//TODO Setup Log Level
```

Comment:
If log-level is not set, will the previous message (inside go func()) be logged properly?

Reply:
As of now, I am getting the logs.
Will check it and update.